config/pmix.m4

```diff
@@ -901,6 +901,10 @@ AC_DEFUN([PMIX_DEFINE_ARGS],[
                         [Whether build should attempt to use dlopen (or
                          similar) to dynamically load components.
                          (default: enabled)])])
+    AS_IF([test "$enable_dlopen" = "unknown"],
```

Comment:
Won't this error out in the OMPI case where we are embedded?

Reply:
Nope, even if PMIx is embedded, it's `configure` is a different process (not like `hwloc` integration view). And I already embedded the patched PMIx and had no issue.

Reply:
sounds good - thx!